### ISSUE TITLE
[attention_ocr] Different results from eval.py and demo_inference.py

### ISSUE THREAD

**Issue by snehakandpal** [tensorflow/models#6498]
### System information
- **What is the top-level directory of the model you are using**: models/research/attention_ocr/python/
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**: No
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Mac os 10.14
- **TensorFlow installed from (source or binary)**: https://www.tensorflow.org
- **TensorFlow version (use command below)**: 1.10
- **Bazel version (if compiling from source)**: NA
- **CUDA/cuDNN version**: NA 
- **GPU model and memory**: NA
- **Exact command to reproduce**:

### Describe the problem
I tried training model with just a few images and the loss after a while stays around 5.5. However, eval.py script run on the same images give accuracy of 1.0 but when I run demo_inference.py on the same images, the predictions are incorrect.

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be an old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.

**Comment by R1234A** [comment]
@snehakandpal Have you got the correct results using demo_inference.py as I am also not getting the correct results. If you have found any solution please post here which will be helpful for me.

Thanks in advance.
